Error during `npm install`

Dependency conflict: our direct dependency `angular/core@18.2.8` depends on `zone.js@"~0.14.10"` which conflicts with our other direct dependency `zone.js@0.15.0`.

Latest angular/core@18.2.9 also uses zone.js@0.14.10, but in the master angular/core bumped zone.js [already](https://github.com/angular/angular/commit/5cc63c2135b78ad9da2364adb9e7563a4947abc5). This would go into angular/core@19.

Output:

```sh
npm ERR! code ERESOLVE
npm ERR! ERESOLVE could not resolve
npm ERR! 
npm ERR! While resolving: @angular/core@18.2.8
npm ERR! Found: zone.js@0.15.0
npm ERR! node_modules/zone.js
npm ERR!   zone.js@"~0.15.0" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer zone.js@"~0.14.10" from @angular/core@18.2.8
npm ERR! node_modules/@angular/core
npm ERR!   @angular/core@"^18.2.8" from the root project
npm ERR!   peer @angular/core@"18.2.8" from @angular/animations@18.2.8
```

--- a/app/package.json
+++ b/app/package.json
@@ -45,8 +45,7 @@
     "rxjs": "~7.8.1",
     "stream": "^0.0.3",
     "stream-browserify": "^3.0.0",
-    "tslib": "^2.7.0",
-    "zone.js": "~0.15.0"
+    "tslib": "^2.7.0"
   },
   "devDependencies": {
     "@angular-devkit/build-angular": "^18.2.8",

